src/lobby.cpp

```diff
@@ -539,6 +539,7 @@ void Lobby::set_player_count(int players_online, int max_players)
                          .arg(QString::number(players_online))
                          .arg(QString::number(max_players));
   ui_player_count->setText(f_string);
+  enable_connect_button();
```

Comment:
Probably not where this belongs. I have an unstaged patch like this:

```patch
diff --git a/src/packet_distribution.cpp b/src/packet_distribution.cpp
index 44e2816..670dab4 100644
--- a/src/packet_distribution.cpp
+++ b/src/packet_distribution.cpp
@@ -157,6 +157,9 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
     s_pv = f_contents.at(0).toInt();
     server_software = f_contents.at(1);
 
+    if (lobby_constructed)
+      w_lobby->enable_connect_button();
+
     send_server_packet(new AOPacket("ID#AO2#" + get_version_string() + "#%"));
   }
   else if (header == "CT") {
```

Probably should have committed that, though I thought the connect button was working fine for me. (Or was it working fine *because* I had the patch? Duh... :facepalm:)

Reply:
Let's see if 3d3de26140041d9e502f71f58af4a1f64615b5f2 does it.

Reply:
Works for me, removing that bit from the PR